alpine-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -76,6 +66,7 @@ http {
 
   more_set_headers "X-Content-Type-Options: nosniff";
   more_set_headers "X-Frame-Options: deny";
+  more_set_headers "Content-Security-Policy: frame-ancestors 'none'"
```

Comment:
@dkubb As I cannot find a definitive / authoritative description about the default behavior of the `Content-Security-Policy` I think we should go with the header as is.

From what I understand: By default everything is forbidden unless allowed by this header. But I cannot determine why we'd need to set `frame-ancestors 'none'" in that case.

And if sending an empty valued header would default to forbid everything (which would be what I wanted for a base repository like this).